internal/controller/standalone_pgadmin/pod.go

```diff
@@ -242,28 +242,52 @@ func podConfigFiles(configmap *corev1.ConfigMap, pgadmin v1beta1.PGAdmin) []core
 }
 
 func startupScript(pgadmin *v1beta1.PGAdmin) []string {
-	// loadServerCommand is a python command leveraging the pgadmin setup.py script
+	// loadServerCommandv7 is a python command leveraging the pgadmin v7 setup.py script
 	// with the `--load-servers` flag to replace the servers registered to the admin user
 	// with the contents of the `settingsClusterMapKey` file
-	var loadServerCommand = fmt.Sprintf(`python3 ${PGADMIN_DIR}/setup.py --load-servers %s/%s --user %s --replace`,
+	var loadServerCommandv7 = fmt.Sprintf(`python3 ${PGADMIN_DIR}/setup.py --load-servers %s/%s --user %s --replace`,
 		configMountPath,
 		clusterFilePath,
 		fmt.Sprintf("admin@%s.%s.svc", pgadmin.Name, pgadmin.Namespace))
 
-	// This script sets up, starts pgadmin, and runs the `loadServerCommand` to register the discovered servers.
+	// loadServerCommandv8 is a python command leveraging the pgadmin v8 setup.py script
+	// with the `load-servers` sub-command to replace the servers registered to the admin user
+	// with the contents of the `settingsClusterMapKey` file
+	var loadServerCommandv8 = fmt.Sprintf(`python3 ${PGADMIN_DIR}/setup.py load-servers %s/%s --user %s --replace`,
+		configMountPath,
+		clusterFilePath,
+		fmt.Sprintf("admin@%s.%s.svc", pgadmin.Name, pgadmin.Namespace))
+
+	// setupCommands (v8 requires the 'setup-db' sub-command)
+	var setupCommandv7 = "python3 ${PGADMIN_DIR}/setup.py"
+	var setupCommandv8 = setupCommandv7 + " setup-db"
+
+	// This script sets up, starts pgadmin, and runs the appropriate `loadServerCommand` to register the discovered servers.
 	var startScript = fmt.Sprintf(`
 PGADMIN_DIR=/usr/local/lib/python3.11/site-packages/pgadmin4
+APP_RELEASE=$(cd $PGADMIN_DIR && python3 -c "import config; print(config.APP_RELEASE)")
 
 echo "Running pgAdmin4 Setup"
-python3 ${PGADMIN_DIR}/setup.py
+if [ $APP_RELEASE -eq 7 ]; then
+%s
```

Comment:
:thinking: Here and below, could we indent the conditional statements? I understand if that breaks the unit test or otherwise causes issues with the script, but if possible I think it would help readability.
```suggestion
	%s
```